handouts/Session_14.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Phylogenies in Computational Historical Linguistics\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "### 1.1 Inner and Outer Language History\n",
+    "The origins of phylogenetics go back to the 19th century and its principles haven't changed much since. Among the lesser known early linguists there is Georg von der Gabelentz (1840-1893). The following description of phylogenetic reconstruction which is based on his words from 1891 may still hold true:\n",
+    "\n",
+    ">This branch of linguistics is at first concerned with the most dry individual facts: Are languages A and related and to what degree? Does this word or word form exist in that language or at that time of language history? How is it pronounced there? What kinds of rules can be found in the sound differences? Is it a native or a loan word? Which words belong to the original lexicon, which have been added later? And so on. All this sounds and really is very dry. The innermost drive of human language, that which otherwise makes comparative linguistics one of the most lively subjects, stands back at first: Merely a few of the field's offshoots wind to the mental and social life of the speakers. A linguist focused on a single language hurries to absorb it into themself whereas a language historian stands outside of it, like an anatomist next to a corpse.\n",
```

Comment:
> A linguist focused on a single language hurries to absorb it into themself whereas a language historian stands outside of it, like an anatomist next to a corpse.

rewrite as

> Linguists focusing on a single language hurry to absorb it whereas language historians stand outside of their research object, like anatomists next to a corpse.

Also: "most dry" -> "driest". But you may consider rephrasing this by writing a text with your own words where you mention this. Translating these ancient quotes by Gabelentz is a bit tricky anyway, so maybe it's not needed.